clients/client/src/main/java/org/projectnessie/client/http/HttpNamespaceClient.java

```diff
@@ -85,4 +86,17 @@ public GetNamespacesResponse getNamespaces(@NotNull MultipleNamespacesParams par
         .get()
         .readEntity(GetNamespacesResponse.class);
   }
+
+  @Override
+  public void updateProperties(
+      NamespaceParams params, NamespacePropertyUpdate namespacePropertyUpdate)
+      throws NessieNamespaceNotFoundException, NessieReferenceNotFoundException {
+    client
+        .newRequest()
+        .path("namespaces/namespace/{ref}/{name}/properties")
```

Comment:
Remote the `/properties` suffix - it's already a POST request, that's different from the other requests:
PUT -> create
POST -> update
DELETE -> delete